src/plugins/data/server/search/opensearch_search/get_default_search_params.ts

```diff
@@ -49,7 +49,7 @@ export async function getDefaultSearchParams(uiSettingsClient: IUiSettingsClient
     maxConcurrentShardRequests:
       maxConcurrentShardRequests > 0 ? maxConcurrentShardRequests : undefined,
     ignoreThrottled,
-    ignoreUnavailable: true, // Don't fail if the index/indices don't exist
+    ignoreUnavailable: false, // Don't fail if the index/indices don't exist
```

Comment:
Need to make a different function to be called for fetching the default params for PIT searches.